ocdsmerge/fixtures/string_list_example.py

```diff
@@ -0,0 +1,57 @@
+releases = [
+    {
+        "ocid": "A",
+        "id": "1",
+        "date": "2014-01-01",
+        "tag": ["award"],
+        "parties": [{
+          "id": "consequat reprehenderit",
+          "name": "sit Lorem irure proident",
+          "roles": ["buyer"]
```

Comment:
For a more robust test, shouldn't we have the test use multiple roles (or more precisely, arrays of roles of different lengths)?